game-app/game-core/src/main/java/games/strategy/triplea/ui/menubar/ExportMenu.java

```diff
@@ -162,14 +162,14 @@ private void createAndSaveStats(final boolean showPhaseStats) {
     if (chooser.showSaveDialog(frame) != JOptionPane.OK_OPTION) {
       return;
     }
+    final var cloneOptions = GameDataManager.Options.builder().withHistory(true).build();
+    final GameData clone = GameDataUtils.cloneGameData(gameData, cloneOptions).orElse(null);
+    if (clone == null) {
+      return;
+    }
     try (PrintWriter writer =
             new PrintWriter(chooser.getSelectedFile(), StandardCharsets.UTF_8.toString());
         GameData.Unlocker ignored = gameData.acquireReadLock()) {
```

Comment:
Why is a lock to gameData held here, shouldn't it use the clone?

Reply:
It still uses gameData though. Agree that it can probably use the clone though.